### PR TITLE
add molecule testing and auto documentation

### PR DESCRIPTION
**Pull request by xoxys**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    actions:
+    - commit
+    - push
+    author_email: "drone@webhippie.de"
+    author_name: Drone
+    branch: master
+    message: "[SKIP CI] update readme"
+    remote: "https://github.com/rolehippie/docker"
+    netrc_machine: github.com
+    netrc_useriname: tboerger
```
Comment (tboerger) [inline .drone.yml]: Typo

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    netrc_password:
+      from_secret: github_token
+  when:
+    ref:
+    - refs/heads/master
+    - refs/tags/**
```
Comment (tboerger) [inline .drone.yml]: I think you could drop the tag event here

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+name: deployment
+
+platform:
+  os: linux
+  arch: amd64
+
+concurrency:
+  limit: 1
+
+workspace:
```
Comment (tboerger) [inline .drone.yml]: Why do we need a workspace here?

Comment (xoxys) [reply .drone.yml]: Because I have not found a better way. Molecule does not find the role otherwise.

**Pull request merged**